src/Elcodi/Bundle/FixturesBoosterBundle/Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -83,13 +83,11 @@ public function __construct(
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         if (empty($this->databaseFilePath)) {
-            parent::execute($input, $output);
-
-            return 1;
+            return parent::execute($input, $output);
```

Comment:
So, that's why my refresh script was failing without errors after fixture loading. Thanks for the fix!
Just a minor thing, CS requires new line before return statements.